templates/default/puma.rb.erb

```diff
@@ -56,6 +56,8 @@ threads <%= @out[:thread_min] %>, <%= @out[:thread_max] %>
 # bind 'ssl://127.0.0.1:9292?key=path_to_key&cert=path_to_cert'
 <% if @webserver == 'apache2' %>
 bind "tcp://127.0.0.1:3000"
+<% elsif @out[:port] %>
+bind "tcp://0.0.0.0:<%= @out[:port] %>"
```

Comment:
Can you switch conditions here, to make this webserver independent? Something like

```ruby
<% if @out[:port] %>
bind "tcp://0.0.0.0:<%= @out[:port] %>"
<% elsif @webserver == 'apache2' %>
bind "tcp://127.0.0.1:3000"
```

Reply:
great suggestion. that'll make this more flexible.